README.md

```diff
@@ -14,6 +14,13 @@ in Bazel, and by the
 [`swift_library` rule](https://github.com/bazelbuild/rules_swift/blob/master/doc/rules.md#swift_library)
 available from [rules_swift](https://github.com/bazelbuild/rules_swift).
 
+The goal of rules_ios has always been to allow seamlessly transitioning a set of targets from being built
```

Comment:
Can you speak to the goals on the Xcode generation side of this repo? There's been a lot interest peaked on it recently and many of the commits here are regarding that

Reply:
I'll leave that to amber to expound on

Reply:
@gyfelton should be able to help with this too while Amber is out